reporter-web-app/src/components/PackageFindingsTable.js

```diff
@@ -19,14 +19,104 @@
 
 import React from 'react';
 import PropTypes from 'prop-types';
-import { Table } from 'antd';
-import { CopyrightOutlined, FileTextOutlined } from '@ant-design/icons';
+import { Table, Tooltip } from 'antd';
+import {
```

Comment:
commit message: The title says "package configuration excludes", but isn't the handling actually for `PathExcludes` in general, that is for `packages` and `projects`?

How about instead: "reporter-web-app: Handle path excludes on finding granularity"
as previously they have been handled only on project granularity.

Reply:
Fixed